habit/src/components/screens/login/index.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
```

Comment:
Move the file to ..src/containers/Login/LoginScreen ?

Reply:
okay